data/json/mapgen/mine/mine_shaft.json

```
@@ -78,6 +78,7 @@
         "=": "t_conveyor",
         "*": "t_railroad_track_small",
         "®": "t_thconc_floor",
+        "№": "t_pillar",
```

```suggestion
        "№": "t_pillar",
```
Something in US-ASCII would probably be preferable, since people's systems are going to differ on how to interpret this (what do you see it as? I see a capital N with a superscripted, underlined o).

---

I'm actively using unicode characters, see #46726 for example, where there are LOTS of them. I know it's making direct editing a bit harder, but there are many advantages of using such characters, so it's fully justified, at least for me.

---

Understand; you did kind of run out of US-ASCII characters for that one...